Runtime/Model/Breadcrumbs/InMemory/BacktraceInMemoryLogManager.cs

```diff
@@ -46,6 +46,15 @@ public string BreadcrumbsFilePath
             }
         }
 
+        /// <summary>
+        /// Archive in memory breadcrumbs. For in-memory breadcrumbs this operation is unsupported.
+        /// </summary>
+        /// <returns>Empty string</returns>
+        public string Archive()
```

Comment:
If you want to make unsupported features, why not introduce a new interface, like `IArchivable`? That way, you don't have to make empty methods that return empty strings, and when you want to use the `Archive` method, you would have to check if the class supports it (`if (instance is IArchivable)`).